sentry/src/main/java/io/sentry/SessionReplayOptions.java

```diff
@@ -0,0 +1,85 @@
+package io.sentry;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+
+public final class SessionReplayOptions {
```

Comment:
@brustolin I've seen you used `SentryReplayOptions` instead of `SessionReplayOptions`, I thought of skipping the "Sentry" part, since it's anyway a nested object under `sentryOptions._experimental` already. But let me know if you want me to change that, I'm fine with either

Reply:
the class name for objc can collide with any framework in the app. That's why we need to have the prefix everywhere for the class name. And SentrySessionReplayOptions looks too big IMO.

Reply:
ah makes sense, I can change that, no problem 👍